strongbox-web-forms/src/main/java/org/carlspring/strongbox/forms/users/AccessModelForm.java

```diff
@@ -31,4 +34,23 @@ public void addRepositoryAccess(RepositoryAccessModelForm repositoryAccess)
         }
         repositoriesAccess.add(repositoryAccess);
     }
+
+    public List<String> getApiAccess()
+    {
+        return apiAccess;
+    }
+
+    public void setApiAccess(List<String> apiAccess)
+    {
+        this.apiAccess = apiAccess;
+    }
+    
+    public void addApiAccess(String privilege) {
```

Comment:
Cosmetic: Braces on new line here and in the `if` statement, please.

Reply:
fixed